source/features/collapse-markdown-sections.tsx

```diff
@@ -44,7 +45,7 @@ void features.add(__filebasename, {
 	include: [
 		pageDetect.isRepoTree,
 		pageDetect.isRepoWiki,
-		() => pageDetect.isSingleFile() && location.pathname.endsWith('.md')
+		() => pageDetect.isSingleFile() && isMarkdownFile()
```

Comment:
It should still not just be limited to markdown files. It's _rendered/markup text files_, almost everywhere in this PR

Reply:
I see, so some of these features are actually mistakenly limited to `.md` files when implemented.
Shy we change their names to something else so indicate they work on more files, not just markdown? Or perhaps a separate PR in the future.